Document conditions in Kubernetes (#47149)
kubernetes/website: content/en/docs/reference/glossary/condition.md

---
term_id: condition
title: Condition
date: 2024-07-12
full_link: /docs/concepts/overview/working-with-objects/conditions/
short_description: >

>     A detailed and structured way to provide the status of Kubernetes objects.

Reviewer: ```suggestion
    A machine-readable way to communicate status for Kubernetes objects.
```

aka:
tags:
- fundamental
---

> Conditions are a detailed and structured way to provide the status of Kubernetes objects. They include fields like Type, Status, Last Transition Time, Reason, and Message. For more details, see [Conditions in Kubernetes](/docs/concepts/overview/working-with-objects/conditions/).
> 
> <!--more-->

Reviewer: ```suggestion
A machine-readable way to communicate Boolean (or unknown) status about an aspect of
a Kubernetes object.

<!--more-->

For more details, see [Conditions in Kubernetes](/docs/concepts/overview/working-with-objects/conditions/).
```
